### PR TITLE
Rifles - Grid inventory adjustments

### PR DESCRIPTION
**Pull request by ProphetCrow**
## About the PR

This is an adjustment for the majority of ranged weapons that are larger than a pistol.
In their current state, **most** firearms, including lasers, consume an ergonomically oppressive and nonsensical 4x4 block, with a handful of notable exceptions that stray from this law.
This PR aims to realistically standardize the weapon sizes with respect to the weapons that already have a defined size (notably, the C20-R)

## Why / Balance
This standardizes the size of firearms. SMGs, such as the C-20r, were already 4x2, so this gets the rest of the weapons up to a similar playing field with sensible size adjustments. Most rifle sized weapons are **longer than they are wide** so a 4x4 does not accurately reflect how they should consume backpack space. 

Notable changes
Most long rifles (LRs) and laser rifles (lasers) are defined to be 5x2, which is slightly longer than the average submachine gun
Shotguns have been updated to match the general size of an LR. Shotguns, such as the Kammerer, now take up 5x2 instead of 5x1. 
Heavier weapons, such as the L6 saw, consume a 5x3 block to indicate its greater bulk
Snipers and bolt action weapons are longer than modern long rifles, so they consume 6x2

Overall, this makes rifles easier to store in backpacks, but there's an existing precedent for estimated weapon sizes when you consider the current backpack size of the Kammerer and the C20-R. 

## Media 

https://discord.com/channels/1263024087647649853/1263031053627428885/1275556221813784576
and below traffic

**Changelog**

:cl: Crow
- tweak: Adjusted grid inventory sizes for rifles

### CODE REVIEW

File: Resources/Prototypes/DeltaV/Entities/Objects/Weapons/Guns/Battery/battery_guns.yml [hunk 1]
```diff
@@ -115,6 +115,10 @@
     price: 750
   - type: StealTarget
     stealGroup: WeaponEnergyGunMultiphase
+  - type: Item
```
Comment (redmushie) [inline Resources/Prototypes/DeltaV/Entities/Objects/Weapons/Guns/Battery/battery_guns.yml:119]: Add Echo marker `# Echo: Decrease size` or similar

File: Resources/Prototypes/Entities/Objects/Weapons/Guns/Battery/battery_guns.yml [hunk 2]
```diff
@@ -346,6 +354,10 @@
   - type: HitscanBatteryAmmoProvider
     proto: RedHeavyLaser
     fireCost: 100
+  - type: Item
```
Comment (redmushie) [inline Resources/Prototypes/Entities/Objects/Weapons/Guns/Battery/battery_guns.yml:359]: Add Echo marker w/ brief description

File: Resources/Prototypes/Entities/Objects/Weapons/Guns/Battery/battery_guns.yml [hunk 3]
```diff
@@ -405,6 +417,10 @@
     steps: 5
     zeroVisible: true
   - type: Appearance
+  - type: Item
```
Comment (redmushie) [inline Resources/Prototypes/Entities/Objects/Weapons/Guns/Battery/battery_guns.yml:422]: Add Echo marker w/ brief description

File: Resources/Prototypes/Entities/Objects/Weapons/Guns/Battery/battery_guns.yml [hunk 4]
```diff
@@ -585,6 +603,10 @@
     price: 750
   - type: StealTarget
     stealGroup: WeaponAntiqueLaser
+  - type: Item
```
Comment (redmushie) [inline Resources/Prototypes/Entities/Objects/Weapons/Guns/Battery/battery_guns.yml:614]: Add Echo marker w/ brief description

File: Resources/Prototypes/Entities/Objects/Weapons/Guns/Shotguns/shotguns.yml [hunk 5]
```diff
@@ -59,6 +61,8 @@
       map: ["enum.GunVisualLayers.Mag"]
   - type: Item
     size: Large
+    shape:
```
Comment (redmushie) [inline Resources/Prototypes/Entities/Objects/Weapons/Guns/Shotguns/shotguns.yml:64]: Add Echo marker w/ brief description

File: Resources/Prototypes/Entities/Objects/Weapons/Guns/Shotguns/shotguns.yml [hunk 6]
```diff
@@ -223,11 +229,17 @@
   components:
   - type: Item
     size: Small
+    shape:
```
Comment (redmushie) [inline Resources/Prototypes/Entities/Objects/Weapons/Guns/Shotguns/shotguns.yml:232]: Add Echo marker w/ brief description

File: Resources/Prototypes/Entities/Objects/Weapons/Guns/Snipers/snipers.yml [hunk 7]
```diff
@@ -73,6 +75,8 @@
   - type: Sharp
   - type: Item
     size: Large
+    shape:
```
Comment (redmushie) [inline Resources/Prototypes/Entities/Objects/Weapons/Guns/Snipers/snipers.yml:78]: Add Echo marker w/ brief description

**Review by redmushie — CHANGES_REQUESTED** [review]

**Comment by ProphetCrow** [comment]
wang-jangled my text editor and managed to get it merged and updated. went back and added echo markers at the requested spots.

**Review by redmushie — APPROVED** [review]

**Pull request merged**